api/rights.proto

```diff
@@ -119,6 +119,10 @@ enum Right {
   RIGHT_GATEWAY_STATUS_READ = 38;
   // The right to view view gateway location.
   RIGHT_GATEWAY_LOCATION_READ = 39;
+  //The right to encrypt and store the a secret value associated with this gateway.
+  RIGHT_GATEWAY_WRITE_SECRET = 57;
+  //The right to decrypt and retrieve the LBS secret value.
+  RIGHT_GATEWAY_READ_SECRET = 58;
```

Comment:
- "the a" secret?
- Or the LBS secret?
- Or plural secrets?
- Spaces after `//`